tests/core/daemon/test_daemon.py

```diff
@@ -169,6 +170,100 @@ async def reader(ws, queue):
     assert blockchain_state_found
 
 
+@pytest.mark.asyncio
+async def test_is_running_wildcard():
+    # Simple class that responds to a poll() call used by WebSocketServer.is_running()
+    @dataclass
+    class Service:
+        running: bool
+
+        def poll(self) -> Optional[int]:
+            return None if self.running else 1
+
+    # Mock daemon server that forwards to WebSocketServer
+    @dataclass
+    class Daemon:
+        # Instance variables used by WebSocketServer.is_running()
+        services: Dict[str, Union[List[Service], Service]]
+        connections: Dict[str, Optional[List[Any]]]
+
+        async def is_running(self, request: Dict[str, Any]) -> Dict[str, Any]:
+            return await WebSocketServer.is_running(self, request)  # type: ignore
```

Comment:
Please provide the full ignore form such as `# type: ignore[no-any-return]` or whatever it is as well as a comment on the previously line either with a link to a bug or PR that would address the need for the ignore or a long-term justification for it.

Reply:
Updated